Assets/Scripts/UI/Menu/MenuManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿public static class MenuManager
```

Comment:
I'm quite happy with this class. I managed to create a class specifically for our implementation of the counter (`RegistrationCounter`), but I also wrapped it in super readable interface (`MenuManager`).